### PR TITLE
Add partial NTLM support to oraclelinux

### PR DESCRIPTION
**Pull request by RDIL**
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [x] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [x] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by TravisEz13** [comment]
Why are we not enabling the tests?

**Comment by RDIL** [comment]
I can try to run them

**Comment by RDIL** [comment]
@TravisEz13 test failed with same reason as amazonlinux

**Comment by TravisEz13** [comment]
Can you build the image locally as see where `gssntlmssp.so` exists?

**Comment by RDIL** [comment]
I can try, my CPU may not be powerful enough though.

File: release/community-stable/oraclelinux/docker/Dockerfile [hunk 1]
```diff
@@ -60,7 +60,8 @@ RUN \
     # clean cached data
     && yum clean all \
     # remove cache folders and files
-    && rm -rf /var/cache/yum
+    && rm -rf /var/cache/yum \
+    && find /. -name 'gssntlmssp.so' -exec echo {} \;
```
Comment (TravisEz13) [inline release/community-stable/oraclelinux/docker/Dockerfile]: To get the logs, add a command that fails after this

**Comment by SteveL-MSFT** [comment]
@PoshChan-Staging Please remind me in 5 minutes

**Comment by SteveL-MSFT** [comment]
@PoshChan-Staging Please rebuild CI

**Comment by SteveL-MSFT** [comment]
@PoshChan-Staging Please remind me in 5 minutes

**Comment by PoshChan-Staging** [comment]
@SteveL-MSFT, reminders are not enabled for this repo.

**Comment by SteveL-MSFT** [comment]
@PoshChan-Staging Please remind me in 5 minutes

**Comment by PoshChan-Staging** [comment]
@SteveL-MSFT, will remind you in 5 minutes

**Comment by SteveL-MSFT** [comment]
@PoshChan-Staging Please rebuild CI

**Comment by PoshChan-Staging** [comment]
@SteveL-MSFT, did not find matching build context: ``; allowed contexts: docker-PR

**Comment by PoshChan-Staging** [comment]
@SteveL-MSFT, this is the reminder you requested 5 minutes ago

**Comment by TravisEz13** [comment]
@PoshChan 	please remind me in 10 hours

**Comment by PoshChan-Staging** [comment]
@TravisEz13, will remind you in 10 hours

**Comment by PoshChan-Staging** [comment]
@TravisEz13, this is the reminder you requested 10 hours ago

**Pull request merged**